pallets/dca/README.md

```diff
@@ -0,0 +1,20 @@
+# DCA pallet
+
+## Overview
+
+A dollar-cost averaging pallet that enables users to perform repeating orders.
+
+When an order is submitted, it will reserve the total amount (budget) specified by the user.
+
+A named reserve is allocated for the reserved amount of all DCA held by each user.
+
+The DCA plan is executed as long as there is remaining balance in the budget.
+
+If a trade fails due to specific errors whitelisted in the pallet config, 
+then retry happens up to the maximum number of retries specified also as config. 
+Once the max number of retries reached, the order is terminated permanently.
+
+If a trade fails due to other kind of errors, the order is terminated permanently without any retry logic.
+
+Orders are executed on block initialize and they are sorted based on randomness derived from relay chain block number. 
+Therefore they cannot be front-ran in the block they are executed.
```

Comment:
that is a strong statement, we only reduce the chance of it happening as transactions can still get ordered to front run you by chance.

Reply:
copied from you :P -> https://www.notion.so/DCA-061a93f912fd43b3a8e3e413abb8afdf?pvs=4#9aa06ddd4dc44ef395828061036e647f

But if it is a strong statement, I can remove it then.

Reply:
unfortunatelly it is not true :) so i would amend it